src/main/java/com/gregtechceu/gtceu/api/blockentity/MetaMachineBlockEntity.java

```diff
@@ -82,6 +88,28 @@ public MultiManagedStorage getRootStorage() {
         return managedStorage;
     }
 
+    @Override
+    public UUID getOwnerUUID() {
+        return this.owner;
+    }
+
+    @Override
+    public void setOwner(UUID uuid, Class<?> ownerType, String ownerName) {
+        this.owner = uuid;
+        this.ownerType = ownerType;
+        this.ownerName = ownerName;
+    }
+
+    @Override
+    public boolean ownerOnline() {
+        return ServerLifecycleHooks.getCurrentServer().getPlayerList().getPlayer(owner) != null;
```

Comment:
This always returns false when a teams mod is used, due to the team's uuid being stored instead. Is that intended?